types/react-navigation/index.d.ts

```diff
@@ -651,3 +652,22 @@ export interface NavigationScreenProps<Params> {
 /**
  * END CUSTOM CONVENIENCE INTERFACES
  */
+
+
+/*
```

Comment:
`react-navigation.js` exports views and header (`Header`, `HeaderTitle`, `HeaderBackTitle`), most of which are not included in current ts definition. I'm not sure if it is better to have it's own section. And I wonder `CUSTOM CONVENIENCE INTERFACES` is customized by who, or just be distinguished with `TypeDefinition.js`.